expats_static/deposit_uob.js

```diff
@@ -0,0 +1,19 @@
+  var requestOptions = {
+    method: 'GET',
+    redirect: 'follow'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Geofflac/DevOps_ShowAndTell/pullRequest?prid=8167659)
```suggestion
    redirect: "follow"
```